libs/utils/utils.js

```diff
@@ -133,6 +133,13 @@ ENVS.local = {
   ...ENVS.stage,
   name: 'local',
 };
+const STAGE_DOMAINS_MAP = {
+  'www.adobe.com': 'www.stage.adobe.com',
+  'blog.adobe.com': 'blog.stage.adobe.com',
+  'business.adobe.com': 'business.stage.adobe.com',
+  'helpx.adobe.com': 'helpx.stage.adobe.com',
+  'news.adobe.com': 'news.stage.adobe.com',
+};
```

Comment:
I'm still not convinced Milo should be maintaining this list. My suggestion would be to ask consumers to add it the way they do `prodDomains`, like Milo [does](https://github.com/adobecom/milo/blob/stage/libs/scripts/scripts.js#L20), for example.

Reply:
Was just about to write the same, milo should be agnostic about who implements it.

Reply:
I was thinking of it as a default value that the consumers can overwrite [through the config they set](https://github.com/adobecom/milo/pull/2361/files#diff-28218f239d8d9c4d41e1924854440b50964619e56058fdd0ae243ff5ff6c616dR225-R227). Anyway, makes sense to keep this completely separate from milo 👍

Reply:
Having sensible defaults was one of my earlier suggestions as well, but we add precious pre LCP bytes to the utils for something that will _never_ be used on production